scripts/modules/aws-mws-common/kms.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_kms_key" "customer_managed_key" {
+}
+
+resource "aws_kms_grant" "databricks-grant" {
+  name = "databricks-grant"
+  key_id  = aws_kms_key.customer_managed_key.key_id
+  // TODO: maybe this should be an env variable
+  grantee_principal = "arn:aws:iam::414351767826:root"
```

Comment:
@stikkireddy should be `databricks_aws_account_id` variable with default

Reply:
added environment variable DATABRICKS_AWS_ACCOUNT_ID to the require_env and module to use it.